packages/gatsby-plugin-gatsby-cloud/src/__tests__/gatsby-browser.js

```diff
@@ -469,6 +427,38 @@ describe(`Preview status indicator`, () => {
           matcherType: `get`,
         })
       })
+
+      // see SKIPPED TEST NOTE
```

Comment:
It looks like you remove the skipped test note in this PR, we probably want to add that back in righT?

Reply:
Oof, it does appear I am sending them on a wild goose skipped note chase. I'll re-add.